cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -28,26 +28,280 @@ Cask Data Application Platform Release Notes
+  applications written in Scala from being deployed.
+
+- `CDAP-5103 <https://issues.cask.co/browse/CDAP-5103>`__ - Fix a problem so that when the
+  schema for a view was not explicitly specified, the view system metadata will include the
+  default schema for the specified format if that is available.
+
+- `CDAP-5131 <https://issues.cask.co/browse/CDAP-5131>`__ - Fixed a problem when filtering
+  plugins by their extension plugin type; filtering by the extensions plugin type was
+  returning extra results for any plugins that did not have an extension.
+
+- `CDAP-5177 <https://issues.cask.co/browse/CDAP-5177>`__ - Fixed a problem with
+  PartitionConsumer not appropriately handling partitions that had been deleted since they
+  were added to the working set.
+
+- `CDAP-5182 <https://issues.cask.co/browse/CDAP-5182>`__ - Fixed a problem preventing
+  programs from administering datasets by adding an API to retrieve the properties that were
+  used to configure (or reconfigure) a dataset.
+
+- `CDAP-5241 <https://issues.cask.co/browse/CDAP-5241>`__ - Fixed a problem with metadata
+  for a dataset not being deleted when a dataset was deleted.
+
+- `CDAP-5267 <https://issues.cask.co/browse/CDAP-5267>`__ - Fixed a problem with the
+  ``PartitionFilter.ALWAYS_MATCH`` not working as an input partition filter.
+  ``PartitionFilter`` is now serialized into one key of the runtime arguments, to support
+  serialization of ``PartitionFilter.ALWAYS_MATCH``. If there are additional fields in the
+  ``PartitionFilter`` that do not exist in the partitioning, the filter will then never match.
+
+- `CDAP-5272 <https://issues.cask.co/browse/CDAP-5272>`__ - Fixed a problem with a null
+  pointer exception when null values were written to a database sink in Cask Hydrator.
+
+- `CDAP-5280 <https://issues.cask.co/browse/CDAP-5280>`__ - Corrected the documentation of
+  the Query HTTP RESTful API for the retrieving of the status of a query.
+
+- `CDAP-5297 <https://issues.cask.co/browse/CDAP-5297>`__ - Fixed a problem with the CDAP
+  UI not supporting pipelines created using previous versions of Cask Hydrator. The UI now
+  shows appropriate information to upgrade the pipeline to be able to view it in the UI.
+
+- `CDAP-5417 <https://issues.cask.co/browse/CDAP-5417>`__ - Fixed an issue with running
+  the CDAP examples in the CDAP SDK under Windows by setting appropriate memory requirements
+  in the ``cdap.bat`` start script.
+
+- `CDAP-5460 <https://issues.cask.co/browse/CDAP-5460>`__ - Fixed a problem with the
+  workflow	Spark programs status not being updated in the CDAP UI on the program list
+  screen when it is run as a part of Workflow.
+
+- `CDAP-5463 <https://issues.cask.co/browse/CDAP-5463>`__ - Fixed an issue when changing
+  the number of instances of a worker or service.
+
+- `CDAP-5465 <https://issues.cask.co/browse/CDAP-5465>`__ - Fixed a problem |---| in the Cask
```

Comment:
this affectsVersion is 3.4.0. So not sure it needs to be in release notes. (unless the affectsVersion on the JIRA is wrong).
@sreevatsanraman ?